build-ps/percona-server.spec

```diff
@@ -680,6 +686,11 @@ fi
   fi
 %endif
 
+%posttrans -n Percona-Server-server%{product_suffix}
+if [ ! -d %{_datadir}/mysql ]; then
```

Comment:
and if it is link?

Reply:
Gotcha, sorry for missing this point. Fixed